docs/README.Linux.md

```diff
@@ -8,14 +8,14 @@ Several distribution **[specific build guides](README.md)** are available.
 ## Table of Contents
 1. **[Document conventions](#1-document-conventions)**
 2. **[Get the source code](#2-get-the-source-code)**
-3. **[Install the required packages](#3-install-the-required-packages)**  
```

Comment:
removing the spaces breaks the layout, see https://github.com/lrusak/xbmc/blob/iso9660-rework/docs/README.Linux.md